gulp/tasks/build.js

```diff
@@ -7,7 +7,6 @@ gulp.task( 'build',
     'styles',
     'scripts',
     'images',
-    'copy',
-    'usage'
+    'copy'
```

Comment:
I disagree. I don't want to have to know different rules for different environments.